packages/terra-tabs/src/Tabs.jsx

```diff
@@ -115,7 +115,7 @@ class Tabs extends React.Component {
     React.Children.forEach(children, child => {
       let content;
       let tabContent;
-      if (child.key === this.state.activeKey) {
+      if (child.key === this.props.activeKey || (!this.props.activeKey && child.key === this.state.activeKey)) {
```

Comment:
Do we need to check `!this.props.activeKey` ?

Reply:
Can we simplify this condition for better readability 
```suggestion
      const activeKey = this.props.activeKey || this.state.activeKey;
      if (child.key === activeKey) {
```

Reply:
Done https://github.com/cerner/terra-framework/pull/1745/commits/9491ceeefbb8edcd28558754f97d71b9291318ec